### DISCUSSION TITLE
nocodb & docsify?

### DISCUSSION THREAD

**Discussion by ghost** [nocodb/nocodb#4679] [Q&A] [answered]
Hi all.

*I really like [docsify](https://docsify.js.org/#/quickstart) and nocodb, would there be any possibility to create a nocodb embed in docsify?*

**Comment by o1lab** [comment, marked as answer]
Embedding iframe view within docsify should be possible already. We can show the [iframe details going forward](https://github.com/nocodb/nocodb/issues/4680).

**Reply by ghost** [reply]
Hi o1lab.

> Embedding iframe view within docsify should be possible already. We can show the https://github.com/nocodb/nocodb/issues/4680.

thank you for feedback.
